src/docs/composing_applications.md

```diff
@@ -97,15 +97,15 @@ An example `package.json` may look like the following:
   },
   "theiaPluginsDir": "plugins",
   "theiaPlugins": {
-    "vscode-builtin-css": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/css-1.39.1-prel.vsix",
-    "vscode-builtin-html": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/html-1.39.1-prel.vsix",
-    "vscode-builtin-javascript": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/javascript-1.39.1-prel.vsix",
-    "vscode-builtin-json": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/json-1.39.1-prel.vsix",
-    "vscode-builtin-markdown": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/markdown-1.39.1-prel.vsix",
-    "vscode-builtin-npm": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/npm-1.39.1-prel.vsix",
-    "vscode-builtin-scss": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/scss-1.39.1-prel.vsix",
-    "vscode-builtin-typescript": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/typescript-1.39.1-prel.vsix",
-    "vscode-builtin-typescript-language-features": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/typescript-language-features-1.39.1-prel.vsix"
+    "vscode-builtin-css": "https://open-vsx.org/api/vscode/css/1.56.2/file/vscode.css-1.56.2.vsix",
+    "vscode-builtin-html": "https://open-vsx.org/api/vscode/html/1.56.2/file/vscode.html-1.56.2.vsix",
+    "vscode-builtin-javascript": "https://open-vsx.org/api/vscode/javascript/1.56.2/file/vscode.javascript-1.56.2.vsix",
+    "vscode-builtin-json": "https://open-vsx.org/api/vscode/json/1.56.2/file/vscode.json-1.56.2.vsix",
+    "vscode-builtin-markdown": "https://open-vsx.org/api/vscode/markdown/1.56.2/file/vscode.markdown-1.56.2.vsix",
+    "vscode-builtin-npm": "https://open-vsx.org/api/vscode/npm/1.56.2/file/vscode.npm-1.56.2.vsix",
+    "vscode-builtin-scss": "https://open-vsx.org/api/vscode/scss/1.56.2/file/vscode.scss-1.56.2.vsix",
+    "vscode-builtin-typescript": "https://open-vsx.org/api/vscode/typescript/1.56.2/file/vscode.typescript-1.56.2.vsix",
+    "vscode-builtin-typescript-language-features": "https://open-vsx.org/api/vscode/typescript-language-features/1.56.2/file/vscode.typescript-language-features-1.56.2.vsix"
```

Comment:
We might want to use a version which is compatible with our default supported vscode API (currently `1.50.0`).
The `vscode-builtins` are a special type of builtin, where their `version` matches an actual vscode version and this feature is used to determine their compatibility.

Additionally:
- installing `vscode-builtins` (as of `eclipse theia 1.14.0`) through the `extensions-view` will fetch a version which matches our API and will not check the `engines.vscode` property like other extensions. Meaning, it will install `1.50.0` if possible and a previous version if not available.
- the `typescript-language-features` version is currently untested (the main repo uses [`1.45.1`](https://github.com/eclipse-theia/theia/blob/master/package.json#L147)) so it might be better to use a version which known to work and not too new (as it might make use of newer LSP features).

cc @marcdumais-work

Reply:
I agree. The simple extensions that only contribute a grammar and snippets, are very likely to work. But the ones that provide a LS are more breakage-prone and so unlikely to work with such a delta. I suggest like vince that we stick to `1.50.0` for now, with the mentioned exception. Soon we can update this example to use the built-in extensions pack, I think. [WIP PR that does this for Blueprint](https://github.com/eclipse-theia/theia-blueprint/pull/93)

Reply:
@vince-fugnitto is it possible yet to request an extension, from `package.json`, using `namespace`.`id`? It would then resolve to the currently supported version, 1.50.0, but without hard-coding it in the example.

Reply:
makes sense! I update the versions and created https://github.com/eclipse-theia/theia-website/issues/166